hiss/application/models.py

```diff
@@ -1,3 +1,5 @@
+import datetime
```

Comment:
Fun fact that I learned last night, we should be using Django's `timezone` package instead of `datetime`. They function similarly, but the `timezone` package is time zone aware, unlike `datetime`.

```suggestion
from django.utils import timezone
```